ome_zarr/writer.py

```diff
@@ -0,0 +1,76 @@
+"""Image writer utility
+
+"""
+import logging
+
+from typing import Any, Dict, Iterator, List, Optional, Type, Union, Tuple, cast
+import zarr
+
+import numpy as np
+
+from .io import parse_url
+from .reader import Node
+
+
+LOGGER = logging.getLogger("ome_zarr.writer")
+
+def write_image(
+        path: str,
+        image: np.ndarray,
+        name: str = "0",
+        group: str = None,
+        chunks: Union[Tuple[int], int] = None,
+        byte_order: Union[str, List[str]] = "tczyx",
+        **metadata):
+    """Writes an image to the zarr store according to ome-zarr specification
+
+    Parameters
+    ----------
+    path: str,
+      a path to the zarr store location
+    image: np.ndarray
+      the image to save
+    group: str, optional
+      the group within the zarr store to store the data in
+    chunks: int or tuple of ints,
+      size of the saved chunks to store the image
+    byte_order: str or list of str, default "tczyx"
+      combination of the letters defining the order
+      in which the dimensions are saved
+    """
+    zarr_location = parse_url(path)
+
+    ## I assume this should be dealt with in
```

Comment:
Agreed. I think what's needed first is a way to pass `mode` through to the `parse_url` and `*Location` classes.